src/users/users.controller.ts

```diff
@@ -275,18 +282,27 @@ export class UsersController {
       };
     }
 
-    const { data, hasNext, hasPrevious } = await this.usersService.list({
+    const {
+      data: records,
+      hasNext,
+      hasPrevious,
+    } = await this.usersService.list({
       after,
       before,
       limit,
       search,
       countryCode,
       eventTypes,
     });
+    const data = [];
+    for (const record of records) {
+      const userPoints = await this.userPointsService.findOrThrow(record.id);
+      data.push(serializedUserFromRecord(record, userPoints));
+    }
```

Comment:
Optionally, If you want a little potential speed boost, you could do something like this.
```suggestion
    const data = await Promise.all(records.map(async (record) => {
        const userPoints = await this.userPointsService.findOrThrow(record.id);
        return serializedUserFromRecord(record, userPoints);
    }));
```